### PR TITLE
Properly serve static files in production with `whitenoise`

### PR DESCRIPTION
**Pull request by mvandenburgh**
This will fix the CSS issues on https://rgd.kitware.watch/admin/.

This PR is largely based on https://github.com/kitware-resonant/django-composed-configuration/blob/master/composed_configuration/_static.py

### CODE REVIEW

File: docker/nginx.json [hunk 1]
```diff
@@ -45,6 +45,11 @@
         "pass": "applications/rdwatch"
       }
     },
+    {
+      "match": {
+        "uri": "/staticfiles/*"
```
Comment (BryonLewis) [inline docker/nginx.json:50]: should this be /static/  The `STATIC_URL` in the settings.py is `static` the `STATIC_ROOT` which I think is the file location is `staticfiles`?

**Review by BryonLewis — CHANGES_REQUESTED** [review]
Just a question on the nginx route matching

**Comment by mvandenburgh** [comment]
This isn't working within docker, marking as draft until I can look into this

**Comment by mvandenburgh** [comment]
Looking back into this, I don't think we even need `whitenoise` here. That package is used to serve static files from Python directly, and is useful for when you're running in an environment such as Heroku. But we're already using nginx, so we can just use that instead. I'll open another PR with that.

**Pull request closed without merging**